docs/rules/jsx-key.md

```diff
@@ -47,6 +61,10 @@ The following patterns are considered warnings:
 data.map(x => <>{x}</>);
 ```
 
+### `checkUniquePropKey` (default: `true`)
+
+When `false` the validation for unique `key` in array literal and sibling JSX elements is disabled.
```

Comment:
to avoid a breaking change, this also needs to default to false, just like checkFragmentShorthand.

Reply:
(i can see an argument for it not being enabled by default, but i don't think it's worth the risk)